graylog2-web-interface/src/views/components/views/ShareViewModal.jsx

```diff
@@ -2,9 +2,9 @@
 import * as React from 'react';
 import { get } from 'lodash';
 
+import { FormGroup, HelpBlock, Radio } from 'components/graylog';
+import BootstrapModalConfirm from 'components/bootstrap/BootstrapModalConfirm';
```

Comment:
I tried to use `import { BootstrapModalConfirm } from  'components/bootstrap'` here, but it results in an interesting test failure:

![image](https://user-images.githubusercontent.com/46300478/75455853-1c114f00-597a-11ea-915b-cfc83371682c.png)

Reply:
I did not expected `BootstrapModalForm.jsx` to be involved.